valexa/ploting/utils.py

```diff
@@ -0,0 +1,14 @@
+from valexa.core.profiles import Profile
+
+
+def profile_to_dict(profile: Profile):
+    p = dict(profile.__dict__)
```

Comment:
I don't think you need to wrap `profile.__dict__` with a `dict()`. Same thing with `p["model"]` below.

Reply:
I think I wanted to make copies of the dicts rather than keeping references, not sure why I only did it halfway. A copy is not really needed since I'm just serializing them, I'll remove it.

Reply:
You can keep the `dict()`. however, are we sure that `dict()` makes a real copy?

Reply:
Yes, but only a shallow copy. For exemple it copies `"hello"` as `"hello"`, but an object as a reference. I kind of only went midway with only `profile`, `model`, `levels` and `series.` In the end it doesn't take care of, in other things, the numpy arrays deep in the ProfileLevel objects, so I think I'll just remove them after all.

Reply:
For your information, everything is an object in python ;) However, you can keep it or remove it, in the end, it is a matter or readability.